libs/cart/src/effects/cart-resolver.effects.ts

```diff
@@ -18,14 +18,7 @@ import { DaffCartNotFoundError } from '../errors/not-found';
 import { DaffStorageServiceError } from '@daffodil/core';
 
 function catchResolveErrors(error: Error) {
```

Comment:
Looks like this should have stuff in it, or should be removed

Reply:
yeah it was leftover. removed